Docs/template/partials/footer.tmpl.partial

```diff
@@ -0,0 +1,14 @@
+{{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}
```

Comment:
This is the original footer which is licensed by MS, I'm not a lawyer, but I don't think we can copyright it :)